static/js/pages/instances/InstanceFormGuided.tsx

```diff
@@ -91,111 +124,169 @@ const InstanceFormGuided: FC = () => {
     }
   };
 
+  const getInstanceCreationObj = (): LxdInstanceCreation => {
+    return (
+      isYamlConfig
```

Comment:
This is confusing, because there are two callers to the getInstanceCreationObj  function and the call from switchToYaml  doesn't actually care about the isYamlConfig. Or it is always false in that case. I think we should split the functionionality to have one function turning the formik values into an object.

The isYamlConfig should actually live in the onSubmit function directly.

Also I'd pass in the formik values to that function. Because when calling from onSubmit, the values passed to onSubmit can differ from formik.values. I.e. the onSubmit should use the values from the argument it receives, not the formik.values.